Intersect.Server/Web/RestApi/Types/Player/LevelChange.cs

```diff
@@ -0,0 +1,6 @@
+namespace Intersect.Server.Web.RestApi.Types.Player;
+
+public partial struct LevelChange
+{
+    public int Level { get; set; }
```

Comment:
Can this be changed to have PreviousLevel and Level (yes I realize this is an api change)

Reply:
If this is both a request and a response, then PreviousLevel would be optional (not needed in a request) but Level would be required in both